webtau-core/src/test/java/org/testingisdocumenting/webtau/persona/PersonaTest.java

```diff
@@ -61,4 +61,14 @@ public void shouldNotAllowNestingPersonas() {
         }).should(throwException("nesting personas is not allowed, active persona id: John, " +
                 "attempted to nest persona id: Bob"));
     }
-}
\ No newline at end of file
+
+    @Test
+    public void cannotCreateAPersonWithSameNameAsDefaultPersona() {
```

Comment:
for consistency, do you think we should add test to `PersonaExtensionsTest` as well? The reason I have some tests in there is to make sure we don't assume that groovy DSL delegates strictly to Java. But not sure it makes sense in this case.